src/python/detectors/notebook_best_practice_violation/notebook_best_practice_violation_noncompliant.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why do we need separate files for compliant and non compliant? Can these be in the same `.py` file?

Reply:
to cover all three types of violation(blank cell, cell limit, misplaced imports) in one file is not possible. Is it ok if we create separate files for each example?